cats/check_clean_arguments.py

```diff
@@ -22,26 +28,29 @@ def validate_jobinfo(jobinfo: str, expected_partition_names):
         "cpus",
         "gpus",
     )
-    info = dict([match.groups() for match in re.finditer(r"(\w+)=(\w+)", jobinfo)])
+    info = dict([match.groups() for match in re.finditer(r"(\w+)=([\w.]+)", jobinfo)])
```

Comment:
Was the previous regexp missing something?

Reply:
Yes, it would silently drop fractional parts, so `cpus=2.5` would be parsed as `{ "cpus": "2" }`. We do not have any floats in the jobinfo string, but may in the future.

Reply:
Nice!